daprdocs/content/en/reference/arguments-annotations-overview.md

```diff
@@ -19,12 +19,12 @@ This table is meant to help users understand the equivalent options for running
 | `--resources-path`  | `--resources-path` | `-d` | not supported | Path for components directory. If empty, components will not be loaded. |
 | `--config`  | `--config` | `-c` | `dapr.io/config` | Tells Dapr which Configuration resource to use |
 | `--control-plane-address` | not supported | | not supported | Address for a Dapr control plane |
-| `--dapr-grpc-port` | `--dapr-grpc-port` | | not supported | gRPC port for the Dapr API to listen on (default "50001") |
-| `--dapr-http-port` | `--dapr-http-port` | | not supported | The HTTP port for the Dapr API |
-| `--dapr-http-max-request-size` | --dapr-http-max-request-size | | `dapr.io/http-max-request-size` | Increasing max size of request body http and grpc servers parameter in MB to handle uploading of big files. Default is `4` MB |
-| `--dapr-http-read-buffer-size` | --dapr-http-read-buffer-size | | `dapr.io/http-read-buffer-size` | Increasing max size of http header read buffer in KB to handle when sending multi-KB headers. The default 4 KB.  When sending bigger than default 4KB http headers, you should set this to a larger value, for example 16 (for 16KB) |
+| `--dapr-grpc-port` | `--dapr-grpc-port` | | `dapr.io/grpc-port` | gRPC port for the Dapr API to listen on (default `50001`) |
```

Comment:
There should be a mention that if overriden, all services in the cluster should use the same gRPC port IMHO